tests/foreman/sys/test_pulp3_filesystem.py

```diff
@@ -96,3 +99,28 @@ def test_pulp_workers_status(target_sat):
     output = target_sat.execute('systemctl status pulpcore-worker* | grep Active')
     result = output.stdout.rstrip().splitlines()
     assert all('Active: active (running)' in r for r in result)
+
+
+@pytest.mark.tier1
+def test_pulp_status(target_sat):
+    """Test pulp status via pulp-cli.
+
+    :id: aa7fda5a-cfa1-4fa1-8714-8d5a2e2f89d9
+
+    :expectedresults: Pulp is just fine.
+    """
+    status = json.loads(target_sat.execute('pulp status').stdout)
+    now = datetime.utcnow()
+    assert status['database_connection']['connected']
+    assert status['redis_connection']['connected']
+
+    workers_beats = [
+        datetime.strptime(worker['last_heartbeat'], '%Y-%m-%dT%H:%M:%S.%fZ')
+        for worker in status['online_workers']
+    ]
+    assert all((now - beat).seconds < 20 for beat in workers_beats)
```

Comment:
Where does the value `20` come from? I wonder if it is worth creating some kind of global variable to store the value in. However, it might cause more pollution to the constants.

Reply:
It comes just from my observations, the heartbeat was never older than 10 seconds during multiple syncs.